src/components/Indicator/Indicator.module.scss

```diff
@@ -0,0 +1,17 @@
+@import "../../styles/themes";
+
+$indicator-size: 8px;
+
+.indicator {
+  width: $indicator-size;
+  height: $indicator-size;
+  border-radius: 50%;
+
+  &.colorPrimary {
```

Comment:
we can convert it to primary instead of colorPrimary and it will be easier to use in the tsx code without using helper funtions